### ISSUE TITLE
npm run dev : Unknown script "undefined".

### ISSUE THREAD

**Issue by nyl-auster** [alidcastano/rogue#10]
Hello here ! 
 when running "npm run dev" i get the following message :
( mac OS Sierra 10.12.6, npm 6.1.0 )

```
> roguetest@1.0.0 dev /Applications/MAMP/htdocs/TEST/roguetest
> rogue

Unknown script "undefined".
Perhaps you need to update renso?
```

My package.json:

```json
{
  "name": "roguetest",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "dev": "rogue",
    "build": "rogue build",
    "start": "rogue start"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "react": "^16.4.1",
    "react-dom": "^16.4.1",
    "react-router-dom": "^4.3.1",
    "rogue": "^0.1.1"
  }
}
```

**Comment by alidcastano** [comment]
Haha that's funny. `renso` was what I was going to name it before `rogue`, forgot to change name in that error.

If you run `npm run dev` it should work; I guess I still haven't made that the default, I'll do so now

Btw, thanks for testing Rogue out and opening up first non-author issue!

**Comment by alidcastano** [comment]
Also, I'm also adding a `/examples` directory now that you can reference

**Comment by nyl-auster** [comment]
To get to the next step, i update my scripts , and i got 

> Error: Cannot find module 'fs-extra'

```json  
{
  "scripts": {
    "dev": "rogue dev"
  }
}
```

**Comment by alidcastano** [comment]
Ok, I ended up making `npm run dev` the command in the documentation rather than relying on defaults

I also added `fs-extra` as a dependency and published new version `0.1.2` if you'd like to try it
